scripts/index.js

```diff
@@ -159,14 +157,28 @@ const frontendStatus = async (dataset,baseUrl,newLine,options) => {
     } catch (e) {
       statuses.graphs = 'NOT OK'
     }
-    
     // tables
     const table = $('.htCore').find('tr').length
     if (table > 2) {
       statuses.tables = 'OK'
     } else {
       statuses.tables = 'NOT OK'
     }
+    // validation status report
+    const validationStatus = $('.alert-danger').find('h3').text()
+    if (validationStatus.length > 10) {
```

Comment:
it is validation error message, I cannot create specific validation for each case. I generalized for all tests.

Reply:
@Mikanebu but what is the point of creating invalid data fail with all kind of invalidity then? it would fail in case of only one of them, right?
In this case I was requesting just to make comment on line 168 probably
```
// checking xxx
```

Reply:
But we need to create specific validation for each case I think. That's all these tests are about, no?